### DISCUSSION TITLE
I have a question about themes

### DISCUSSION THREAD

**Discussion by tiramdasg** [alshedivat/al-folio#2457] [Q&A] [answered]
In `config.yml` you mentioned the link: https://github.com/anuraghazra/github-readme-stats/blob/master/themes/README.md

so can i change:

repo_theme_light: rose 
repo_theme_dark: date_night

the variables in config.yml like this without touching the scss files?

**Comment by george-gca** [comment, marked as answer]
Yes, but these only work for the respective cards generated by [github-readme-stats](https://github.com/anuraghazra/github-readme-stats/blob/master/themes/README.md).
